### PR TITLE
34432 sr only headings

### PR DESCRIPTION
**Pull request by tmitchellgcio**
## Description
Makes the headers h3s and changes the wording to spec

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Testing done
manual testing with SR

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx [hunk 1]
```diff
@@ -158,8 +163,8 @@ const hasAppealsErrorSelector = state => {
 const mapStateToProps = state => {
   const claimsState = state.disability.status;
   const claimsV2Root = claimsState.claimsV2;
-  const appealsLoading = claimsV2Root.appealsLoading;
-  const claimsLoading = claimsV2Root.claimsLoading;
+  const { appealsLoading } = claimsV2Root;
```
Comment (jerekshoe) [inline src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx:166]: I’m sure this change is just to appease the linter, but it should probably be
```
const { appealsLoading, claimsLoading } = claimsV2Root;
```

Comment (tmitchellgcio) [reply src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx:166]: This was an auto modification from my webstorm. Sorry I will get it fixed.

Comment (jerekshoe) [reply src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx:166]: No worries, I figured it was auto-fixed 👍

**Review by jerekshoe — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]

**Comment by adamwhitlock1** [comment]
Look like a couple conflicts still, but otherwise LGTM

**Comment by tmitchellgcio** [comment]
This last file I do not know how it got in here. I'm going to undo that or create a new PR

**Pull request closed without merging**